Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -320,19 +354,40 @@ protected virtual void OnBeforeRender()
             }
         }
 
-        protected virtual void SetLocalTransform(Vector3 newPosition, Quaternion newRotation)
+        protected virtual void SetLocalTransform(Vector3 newPosition, Quaternion newRotation, bool hasTrackingStateFlagInfo, PoseDataFlags trackingStateFlags)
```

Comment:
This is a `protected virtual` method, so you can't just update the signature without it being a breaking change for existing users since their code wouldn't compile.

Reply:
good call. adding a facade call through.